FormSchemas/jsonschema.json

```diff
@@ -122,6 +183,130 @@
       "items": {
         "type": "string"
       }
+    },
+    "item_assets": {
+      "type": "object",
+      "title": "Item Assets",
+      "properties": {
+      "cog_default": {
+        "type": "object",
+        "properties": {
+          "type": {
+            "type": "string",
+            "default":"image/tiff; application=geotiff; profile=cloud-optimized"
+          },
+          "roles": {
+            "type": "array",
+            "items": [
+              {
+                "type": "string"
+              }
+            ],
+            "default":[
+              "data",
+              "layer"
+            ]
+          },
+          "title": {
+            "type": "string",
+            "default":"Default COG Layer"
+          },
+          "description": {
+            "type": "string",
+            "default":"Cloud optimized default layer to display on map"
+          }
+        },
+        "required": [
+          "type",
+          "roles",
+          "title",
+          "description"
+        ]
+      }
+    },
+    "required": [
+      "cog_default"
+      ]
+    },
+    "renders": {
+      "type": "object",
+      "title": "Renders"
+    },
+    "providers": {
+      "title": "Providers",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "name"
+        ],
+        "properties": {
+          "name": {
+            "title": "Organization name",
+            "type": "string",
+            "minLength": 1
+          },
+          "description": {
+            "title": "Organization description",
+            "type": "string"
+          },
+          "roles": {
+            "title": "Organization roles",
+            "type": "array",
+            "items": {
+              "type": "string",
+              "enum": [
+                "producer",
+                "licensor",
+                "processor",
+                "host"
+              ]
+            }
+          },
+          "url": {
+            "title": "Organization homepage",
+            "type": "string",
+            "format": "iri"
+          }
+        }
+      }
+    },
+    "assets": {
+      "type": "object",
+      "properties": {
+        "thumbnail": {
```

Comment:
is thumbnail really the only property for the assets object?

Reply:
There could be others, but thumbnails is the only we use in the catalog right now.